### ISSUE TITLE
Equações em LaTeX não estão sendo convertidas em texto

### ISSUE THREAD

**Issue by RPostalli** [scieloorg/opac#1160]
Equações em LaTeX não estão sendo convertidas em texto: 

Ver artigo da neco v28n2: https://new.scielo.br/article/neco/2018.v28n2/385-416/pt/

![image](https://user-images.githubusercontent.com/6084959/51045853-c1693800-15ab-11e9-945b-edce6d25b4a0.png)

Deveria aparecer como:  http://www.scielo.br/scielo.php?script=sci_arttext&pid=S0103-63512018000200385&lng=pt&nrm=iso&tlng=pt

![image](https://user-images.githubusercontent.com/6084959/51045905-e1006080-15ab-11e9-93ac-daeed1507059.png)

No XML: 

```
<inline-formula>
							<alternatives>
								<tex-math id="i01"> \documentclass {article} \usepackage{wasysym} \usepackage[substack]{amsmath} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage[mathscr]{eucal} \usepackage{mathrsfs} \usepackage{pmc} \usepackage[Euler]{upgreek} \pagestyle{empty} \oddsidemargin -1.0in \begin{document} \[${{\mathcal{L}}_{1}}\left( f,g \right)=\frac{1}{2}\underset{{{l}_{1}}\ldots {{l}_{k}}}{\mathop \sum }\,\left| {{f}_{{{l}_{1}}\ldots {{l}_{k}}}}-{{g}_{{{l}_{1}}\ldots {{l}_{k}}}} \right| \] \end{document} </tex-math>
								<graphic xlink:href="1980-5381-neco-28-02-385-i001.svg"/>
							</alternatives>
						</inline-formula>
```
e

```
<alternatives>
								<tex-math id="i02"> \documentclass {article} \usepackage{wasysym} \usepackage[substack]{amsmath} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage[mathscr]{eucal} \usepackage{mathrsfs} \usepackage{pmc} \usepackage[Euler]{upgreek} \pagestyle{empty} \oddsidemargin -1.0in \begin{document} \[ {{\mathcal{L}}_{1}}\left( {{f}^{m}},~{{g}^{m}} \right)\le {{\mathcal{L}}_{1}}\left( f,g \right) \] \end{document} </tex-math>
								<graphic xlink:href="1980-5381-neco-28-02-385-i002.svg"/>
							</alternatives>
						</inline-formula>
```

**Comment by joffilyfe** [comment]
@RPostalli nós já detectamos a causa do problema relacionado a este issue contudo também observamos um comportamento estranho no XML deste artigo.

Se você observar no *site vintage* [1] a **hipótese 1** também possui problemas de renderização, o que torna sua leitura impossível. A priori nossa análise nos leva a crer que houve um erro no XML, onde os delimitadores da equação LaTex não foram respeitados [2] causando o então problema detectado.

Também detectamos que houve um problema na ordenação do **label** da fórmula, onde neste caso específico da **hipótese** deveria vir antes da fórmula [Ver nos anexos]

## Referências

[[1] - Site vintage](http://www.scielo.br/scielo.php?script=sci_arttext&pid=S0103-63512018000200385&lng=en&nrm=iso&tlng=pt)
[[2] - Delimitadores LaTex](http://docs.mathjax.org/en/latest/tex.html#tex-and-latex-math-delimiters)

### Anexos
```
<disp-formula id="e10"><alternatives><tex-math id="tx10">\documentclass {article} \usepackage{wasysym} \usepackage[substack]{amsmath} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage[mathscr]{eucal} \usepackage{mathrsfs} \usepackage{pmc} \usepackage[Euler]{upgreek} \pagestyle{empty} \oddsidemargin -1.0in \begin{document} \[{{Y}_{i}}\left( 1 \right),~{{Y}_{i}}\left( 0 \right)~\bot {{T}_{i}}|{{X}_{i}}$ </tex-math><graphic xlink:href="/media/assets/neco/v28n2/1980-5381-neco-28-02-385-ge10.svg"/></alternatives><label>Hipótese 1:</label></disp-formula>

**Comment by RPostalli** [comment]
@joffilyfe foi testado o problema e conseguimos corrigir.
Vamos documentar no SPS exemplo de equação em Latex em inline-formula.
Obrigada

**Comment by cinthia-vieira** [comment]
Prezados

Em testes recentes, notei que a renderização de equações latex não funciona no navegador Mozila Firefox, dando erros semelhantes aos apresentados anteriormente.

Estou reabrindo o ticket para que uma melhor investigação possa ser realizada.

**Comment by robertatakenaka** [comment]
@RPostalli isso ocorre porque há erro no conteúdo da tag `tex-math`, sendo assim não irá renderizar corretamente. Tem que ser solicitado correção. 
Exemplo: https://arachnoid.com/latex/ (um teste de renderizador de LaTeX, há outros disponíveis na Internet)
Colei uma das equações e olha o resultado 

<img width="1112" alt="Captura de Tela 2022-04-14 às 11 54 28" src="https://user-images.githubusercontent.com/505143/163416827-f9218a89-688b-4d2b-afb6-6e0c3c4ce319.png">

**Comment by RPostalli** [comment]
@robertatakenaka como o site antigo conseguia renderizar esse código Latex corretamente e o site novo não:
https://old.scielo.br/scielo.php?script=sci_arttext&pid=S0103-63512018000200385&lng=en&nrm=iso&tlng=pt

**Comment by cinthia-vieira** [comment]
@robertatakenaka Então... Eu testei esse renderizador e nem mesmo os LaTeX da própria documentação LaTeX estão sendo renderizados...

Veja um exemplo:
Trecho utilizado para realização do teste:
![image](https://user-images.githubusercontent.com/27728729/163429226-12aba95a-790e-4f04-ba90-7fc44e157ce1.png)

Resultado do teste:
![image](https://user-images.githubusercontent.com/27728729/163429415-4f9bfc28-5f23-44c1-9ee6-ba63ed3429eb.png)

O exemplo foi retirado do WikiBooks [LaTeX Basics](https://en.wikibooks.org/wiki/LaTeX/Basics).

Vc teria outro renderizador pra nos indicar? Porque creio que todos os artigos em LaTeX do site estão nesse padrão com documentclass etc, o que significa que teríamos de reprocessar todos os artigos utilizando <tex-math> da plataforma SciELO...

**Comment by robertatakenaka** [comment]
@cinthia-vieira isso não foi uma indicação, foi resultado de uma busca e peguei um deles aleatoriamente

**Comment by robertatakenaka** [comment]
@cinthia-vieira @RPostalli como vocês disseram que funcionou algum dia, vou reavaliar
